RocketChat/Rocket.Chat
# [FIX] Omnichannel Current chats pagination not working

## Proposed changes (including videos or screenshots)





## Issue(s)

Omnichannel - Current Chats not refreshing while clicking on pagination
## Steps to test or reproduce

Steps to reproduce:
1. Go to Omnichannel => Current Chats
2. Add some filter or just Clear all Filters to see all the chats
3. Click to navigate between the pages

Expected Behavior:
After navigating between the pages it is supposed to refresh and display different chats

Actual Behavior:
Even clicking on any page option it doesn't refresh and the chats remain the same of the first page 
## Further comments
